governance/tech-lead-proposal-process.md

```diff
@@ -1,15 +1,15 @@
 <!-- cSpell:ignore chairpersonship markdownlint -->
 # Environmental Sustainability TAG Technical Lead Proposal Process
 
-Environmental Sustainability Technical Leads operate as a team alongside the Co-Chairs.
-The Co-Chair team seeks enhance the leadership capabilities of the existing co-chairs across the key qualities of industry experience, hands-on cloud-native and ESG experience, as well as administrative experience needed to assist in the execution of the various projects and efforts that make the Environmental Sustainability TAG so successful.
+TAG Environmental Sustainability Technical Leads operate as a team alongside the TAG co-chairs.
+The Technical Leads team seeks to enhance the leadership capabilities of the existing co-chairs across the key qualities of industry experience, hands-on cloud-native, ESG experience (Environmental, Social, and Governance) as well as administrative experience needed to assist in the execution of the various projects and efforts.
```

Comment:
More active wording, maybe?  
The Technical Lead Team enhances the leadership...